screens/feed/feed.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import { Text, TextInput, TouchableOpacity, SafeAreaView, ScrollView, RefreshControl } from 'react-native';
 import { fetchMovies } from './../../api';
-import MoviesList from './../../components/moviesList';
+import MoviesList from '../../components/index';
```

Comment:
it's not gonna work. Index is not exporting anything here. Besides it must be `../../components`